app/javascript/dashboard/components/ChatList.vue

```diff
@@ -773,6 +791,74 @@ export default {
     onContextMenuToggle(state) {
       this.isContextMenuOpen = state;
     },
+    initializeExistingFilterToModal() {
```

Comment:
Can we move this function to `app/javascript/shared/mixins/filterMixin.js`

Reply:
Done https://github.com/chatwoot/chatwoot/pull/6310/commits/d8dfe7e5b92e12e9810a4f00a9402602e21baacb